qemu/tests/cfg/hotplug_mem.cfg

```diff
@@ -80,6 +94,18 @@
                     #    target_hugepages = (size_mem / page_size) * 2 + 10
                     target_hugepages = 2058
                     expected_hugepage_size = 2048
+                    variants:
```

Comment:
Why is this variants only applied for `2M_hugepage`?

Reply:
@luckyh Actually it can be applied to every hugepage size.  I thought one test would be OK.   And I discussed with @MiriamDeng , we come to an agreement that we don't specify the hugepage size here anymore,  since they can't be run in one job.  Now just run it according to the current hugepage size on host. Thanks!